website/www/site/content/en/contribute/_index.md

```diff
@@ -73,20 +73,33 @@ detail.
 
 ### Prerequisites
 
- - a GitHub account
- - a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
- - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
-   changes locally
- - [Go](https://golang.org) 1.12 or later installed for Go SDK development
+ - A GitHub account.
+ - A Linux, macOS, or Microsoft Windows development environment
+      - Required *only* For Windows:
```

Comment:
I thought Java and Docker were required for all operating systems? E.g. the manual steps below include installing JAva 8.

Reply:
Sure. To keep consistency, we left the prerequisites as they used to be. Submitted new commit with corrections.